packages/sources/refund.move

```diff
@@ -442,6 +448,14 @@ module refund::refund {
         let timeout_ts = *option::borrow(&pool.timeout_ts);
         assert_claim_phase_time(timeout_ts, clock);
 
+        let current_ts = clock::timestamp_ms(clock);
+
+        // We check if there's at least 3 days difference between the clock and the timeout.
+        // If not, we bump the timeout_ts to correspond to a least the minimum period
+        if (timeout_ts - current_ts >= MIN_CLAIM_PERIOD_MS) {
```

Comment:
Don't we want the reverse check? 
`if (timeout_ts - current_ts < MIN_CLAIM_PERIOD_MS) { ... }`?

Reply:
That's right. Changed it